packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -68,7 +68,8 @@ const ContentContainer = ({
     customProps.className,
   ]);
 
-  const background = backgroundColor || 'light';
+  // border will only be visible when setFocusOnContainer is set for content-container
+  const background = setFocusOnContainer && (backgroundColor || 'light');
```

Comment:
what does this string 'light' signifies ?

Reply:
It is used to default the background color to light if the backgroundColor prop is not set

Reply:
can you add to constant. can we keep the name as lightbackground ?

Reply:
We need to add a WDIO test here to ensure on performing click on a container focus is not added !!

Reply:
Updated